libs/dotnet-sdk/FeatureboardHttpClient.cs

```diff
@@ -0,0 +1,50 @@
+using Featureboard.DotnetSdk.Models;
+using Microsoft.Extensions.Logging;
+using System.Net;
+using System.Net.Http.Json;
+
+namespace Featureboard.DotnetSdk;
+
+internal class FeatureboardHttpClient : IFeatureboardHttpClient
+{
+  private readonly IHttpClientFactory _httpClientFactory;
+  private readonly ILogger<FeatureboardHttpClient> _logger;
+
+  public FeatureboardHttpClient(IHttpClientFactory httpClientFactory, ILogger<FeatureboardHttpClient> logger)
+  {
+    _httpClientFactory = httpClientFactory;
+    _logger = logger;
+  }
+
+  public async Task<(Dictionary<string, FeatureConfiguration>? features, string? lastModified)> FetchUpdates(string? lastModified, CancellationToken cancellationToken)
```

Comment:
Suggest making `lastModified` a `DateTimeOffset?`

Reply:
It comes from the api and gets sent back to the api as a string.
If we did this we would have to care about it's formatting.

Reply:
It will be a standard ISO date afaik

Reply:
Arr I see what you mean now with the know headers.